src/PostageStamp.sol

```diff
@@ -93,6 +102,9 @@ contract PostageStamp is AccessControl, Pausable {
 
     // Lottery pot at last update.
     uint256 public pot;
+    
+    // blocks in 24 hours ~ 24 * 60 * 60 / 5 = 17280 
+    uint256 public minimumValidityBlocks = 17280;
 
```

Comment:
Isn't this a bit hard-coded for the gnosis/xdai chain?   This will be exceedingly (2.4*) long on the goerli test chain.

Reply:
This could be solved by:
1. Making the variable immutable, and then specifically passed through the constructor at contract deployment; or
2. Given that the nature of this is to determine a length in blocks, and we know that there are deployments to goerli, and gnosis chain, one could:

  a. Use assembly to retrieve the chain id.
  b. Based on the chain id, do the math and allocate to the immutable public variable during the constructor. eg:
  
  ```solidity
      constructor(address _bzzToken, uint8 _minimumBucketDepth) {
        bzzToken = _bzzToken;
        minimumBucketDepth = _minimumBucketDepth;
        _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
        _setupRole(PAUSER_ROLE, msg.sender);
        
        // Set chain-specific var
        uint256 chainId;
        assembly {
          chainId := chainid()
        }
        
        if (chainId == 5) minimumValidityBlocks = 24 hours / 12;
        if (chainId == 100) minimumValidityBlocks = 24 hours / 5;
    }
  ```
  
Given the low cost of deployments on Gnosis Chain, and zero cost on Goerli, this does present a way to do the configuration automatically and keep a consistent schelling game between testnet and mainnet (eg: a slow disk / IO on a node may result in a working schelling game on testnet due to the longer block time, but won't on gnosis chain due to the shorter block time).